src/prefect/cli/profile.py

```diff
@@ -191,7 +192,7 @@ def delete(name: str):
     profiles.remove_profile(name)
 
     verb = "Removed"
-    if name == "default":
+    if name == "ephemeral":
```

Comment:
what's going on with this change? I'd like to avoid any special logic based on specific names

Reply:
`default` was previously the "default" profile that we had special logic for in a couple places. I should be able to totally remove this if we don't want any special treatment for any profiles